.github/workflows/dev.yaml

```
@@ -189,16 +189,16 @@ jobs:
           repository_url: InjectiveLabs/injective-admin-ui
           repository_branch: dev
 
-  package-bump-authz-ui:
-    name: 'Package bump Authz UI'
+  package-bump-do-ui:
+    name: 'Package bump DO UI'
     runs-on: ubuntu-latest
     needs: package-bump-layer
     if: ${{ always() && (needs.publish.result == 'success' || github.event_name == 'workflow_dispatch') }}
```

actionlint (1.7.8)

property "publish" is not defined in object type {package-bump-layer: {outputs: {}; result: string}}

(expression)